lib/policyEvaluator/evaluator.js

```diff
@@ -269,4 +269,6 @@ evaluators.evaluateAllPolicies = (requestContext, allPolicies, log) => {
     return verdict;
 };
 
+evaluators.meetConditions = meetConditions;
```

Comment:
Just a small suggestion, but maybe keep the same format as used for other methods like `evaluateAllPolicies`?